File: app/src/main/java/fr/feepin/justchat/GridViewRoomItem.java
package fr.feepin.justchat;

import android.content.Context;
import android.util.AttributeSet;

import androidx.constraintlayout.widget.ConstraintLayout;

public class GridViewRoomItem extends ConstraintLayout {

    public GridViewRoomItem(Context context) {
        super(context);
    }

    public GridViewRoomItem(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public GridViewRoomItem(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
        super.onMeasure(widthMeasureSpec, widthMeasureSpec);

    }
}

File: app/src/main/java/fr/feepin/justchat/CustomViews/GridViewRoomItem.kt
package fr.feepin.justchat.CustomViews

import android.content.Context
import androidx.constraintlayout.widget.ConstraintLayout

class GridViewRoomItem(context: Context) : ConstraintLayout(context){

    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
        super.onMeasure(widthMeasureSpec, widthMeasureSpec)
    }

}